phpcr-shell/querying.rst

```diff
@@ -196,7 +196,7 @@ Set the value of ``a.title`` to the value of the property :
 
 .. code-block:: bash
 
-    PHPCRSH> UPDATE [nt:unstructured] AS a SET a.title = expr('row.getNode().getProperty('some_property')')
+    PHPCRSH> UPDATE [nt:unstructured] AS a SET a.title = expr('row.getNode().getPropertyValue("some_property")')
```

Comment:
this needs double quotes for sure. does it need to be getPropery or would getPropertyValue (as below) be the correct thing?